tests/test_downstream_issue.py

```diff
@@ -1709,3 +1722,27 @@ def test_update_github_project_fields_priority(self, mock_client):
         d._update_github_project_fields(mock_client, self.mock_downstream, self.mock_issue,
                                   github_project_fields, self.mock_config)
         self.mock_downstream.update.assert_called_with({'priority': {'name': 'Critical'}})
+
+    @mock.patch('jira.client.JIRA')
+    def test_update_github_project_fields_priority_bad(self, mock_client):
+        """This function tests `_update_github_project_fields` with
+        a bad priority value.
+        """
+        github_project_fields = {
+            "priority": {
+                "gh_field": "Priority",
+                "options": {
+                    "P0": "Blocker",
+                    "P1": "Critical",
+                    "P2": "Major",
+                    "P3": "Minor",
+                    "P4": "Optional",
+                    "P5": "Trivial"
+                }}}
+        for bad_pv in [None, '', 'bad_value']:
```

Comment:
Bad value in this case means an unexpected mapping (either GH Project value or Jira value that don't exist in the field list)? If it doesn't have a value at all (does that show up as Null? Empty?), we skip that field by default?

Reply:
I believe that `'bad_value'` should be slotted in where we would expect to find, e.g., `'P5'`, and it would result in a failure-to-map result, which I think is analogous to `None` (but it could be `''`) -- that is, all three values should have the same result, but they represent different initial conditions -- and they each should result in skipping the attempt to update the downstream issue.